### PR TITLE
actual vk api fixes

### PR DESCRIPTION
**Pull request by alexus1024**
users.info version is now requied
user id is passed as id, not uid

### CODE REVIEW

File: src/Owin.Security.Providers.VKontakte/Owin.Security.Providers.VKontakte.csproj [hunk 1]
```diff
@@ -33,19 +33,19 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.Owin, Version=3.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Owin.3.0.1\lib\net45\Microsoft.Owin.dll</HintPath>
+      <HintPath>$(SolutionDir)\packages\Microsoft.Owin.3.0.1\lib\net45\Microsoft.Owin.dll</HintPath>
       <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Owin.Security, Version=3.0.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Owin.Security.3.0.1\lib\net45\Microsoft.Owin.Security.dll</HintPath>
+      <HintPath>$(SolutionDir)\packages\Microsoft.Owin.Security.3.0.1\lib\net45\Microsoft.Owin.Security.dll</HintPath>
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.VKontakte/Owin.Security.Providers.VKontakte.csproj:40]: please undo this, and use the absolute path notation

**Comment by TerribleDev** [comment]
Hi, @alexus1024 I'm assuming that PR #228 will fix this issue as well, and I'm going with that one. Do you have any objections/concerns?

**Comment by alexus1024** [comment]
Yes,  #228 is ok too. I can use it.

**Comment by TerribleDev** [comment]
@alexus1024 please see #230

**Pull request closed without merging**